test/util/framework/per_test_framework.go

```diff
@@ -702,7 +702,11 @@ func (tc *perItOrDescribeTestContext) commitTimingMetadata(ctx context.Context)
 	hash := sha256.New()
 	hash.Write(encodedIdentifier)
 	hashBytes := hash.Sum(nil)
-	output := filepath.Join(tc.perBinaryInvocationTestContext.artifactDir, fmt.Sprintf("timing-metadata-%s.yaml", hex.EncodeToString(hashBytes)))
+	output := filepath.Join(tc.perBinaryInvocationTestContext.sharedDir, "test-timing", fmt.Sprintf("timing-metadata-%s.yaml", hex.EncodeToString(hashBytes)))
```

Comment:
please write to both ARTIFACT_DIR and SHARED_DIR.  I can accept a followup if you like.